lib/bolt/outputter/rainbow.rb

```diff
@@ -86,6 +86,13 @@ def print_summary(results, elapsed_time = nil)
         total_msg << " in #{duration_to_string(elapsed_time)}" unless elapsed_time.nil?
         @stream.puts colorize(:rainbow, total_msg)
       end
+
```

Comment:
Do we want to override `print_topics` too? And `print_message` if `bolt guide invalidtopic` should be rainbow too. I think it's also fine to just leave this as is.